docker/seahorn-full-size-rel.Dockerfile

```diff
@@ -0,0 +1,53 @@
+#
+# Dockerfile for SeaHorn
+# This produces package in /seahorn/build
+# Arguments:
+#  - UBUNTU:     trusty, xenial
+#  - BUILD_TYPE: debug, release
+#  - TRAVIS
+#
+
+ARG UBUNTU
+
+# Pull base image.
+FROM seahorn/seahorn-build:$UBUNTU
+
+ARG TRAVIS
+
+# Fetch SeaHorn (just copy on Travis)
+COPY . /seahorn
+RUN if [ "$TRAVIS" = "" ] ; \
+      then git fetch --depth=10 && git reset --hard origin/master ; \
+    fi
+RUN mkdir -p /seahorn/build
+WORKDIR /seahorn/build
+
+ARG BUILD_TYPE
+
+# Build configuration.
+RUN cmake -GNinja \
+          -DCMAKE_BUILD_TYPE=$BUILD_TYPE \ 
+          -DBOOST_ROOT=/deps/boost \
+          -DZ3_ROOT=/deps/z3 \
+          -DLLVM_DIR=/deps/LLVM-3.8.1-Linux/share/llvm/cmake \
+          -DCMAKE_INSTALL_PREFIX=run \
+          -DCMAKE_CXX_COMPILER=g++-5 \
+          -DCPACK_GENERATOR="TGZ" \
+          -DCMAKE_EXPORT_COMPILE_COMMANDS=1 \
+          ../
+
+RUN cmake --build . --target extra  && cmake ..
+RUN cmake --build . --target crab  && cmake ..
+RUN cmake --build . --target install
+RUN cmake --build . --target package
+
+ENV PATH "/seahorn/build/run/bin:$PATH"
+
+WORKDIR /seahorn
+
+RUN echo '#!/bin/sh' > /tmp/cpy.sh && \ 
+    echo 'cp /seahorn/build/*.tar.gz /host/ && bash' >> /tmp/cpy.sh && \
```

Comment:
what is the benefit of `bash` at the end of `cpy`?

Rename `cpy.sh` to be more descriptive (useful when looking at container history)

Reply:
I'm not sure about the `&& bash`, since it was in the original Dockerfile. It's probably not necessary, so I'll remove it and see what happens.

Reply:
@kuhar : you might know the answer to this one

Reply:
The idea was that it that the container always produces the binary and copies it outside, but it also lets you play with seahorn inside the container.